068-performance-testing.md

```diff
@@ -0,0 +1,321 @@
+Timestamp: 2024-03-16-13-57-59-797, Values: [6.12]
+Timestamp: 2024-03-16-13-58-05-067, Values: [6.19]
+```
+
+And these values/data could be used in the next phase (i.e., analyzing and reporting).
+
+### Analyze and report 
+
+We propose, that a `Reporter.class`, which is responsible for retrieving metrics from `<module-name>/target/performance` directory. 
+Reporter will generate a table of all related metrics to that specific test case.
+For instance for Topic Operator performance test case such report could look like this:
+
+| Experiment | Number of Topics | Creation Time (s) | Deletion Time (s) | Total Test Time (s) | STRIMZI_MAX_BATCH_SIZE | MAX_BATCH_LINGER_MS | Reconciliation Max Duration (s) | Max Batch Size | Max Queue Size | UTO Event Queue Time (s) | Describe Configs Max Duration (s) | Create Topics Max Duration (s) | Reconciliations Max Duration (s) | Update Status Max Duration (s) | Max System Load Average 1m Per Core (%) | Max JVM Memory Used (MBs) |
+|------------|------------------|-------------------|-------------------|---------------------|------------------------|---------------------|---------------------------------|----------------|----------------|--------------------------|-----------------------------------|--------------------------------|----------------------------------|--------------------------------|-----------------------------------------|---------------------------|
+| 1          | 500              | 80.582            | 137.925           | 218.507             | 500                    | 30000               | 0.620                           | 364.000        | 261.000        | 598.308                  | 0.234549834                       | 0.479047522                    | 0.620329716                      | 0.107349124                    | 17.9%                                   | 181.68                    |
+| 2          | 500              | 84.553            | 169.537           | 254.090             | 1000                   | 30000               | 7.281                           | 385.000        | 324.000        | 3091.712                 | 0.237088658                       | 6.530933468                    | 7.281358907                      | 0.307568988                    | 22.8%                                   | 172.74                    |
+| ...        | ...              | ...               | ...               | ...                 | ...                    | ...                 | ...                             | ...            | ...            | ...                      | ...                               | ...                            | ...                              | ...                            | ...                                     | ...                       |
+
+We could use such report as a `post-processing` phase after test execution and all needed metrics were gathered from Pods.
+
+### CI: Testing farm
+
+We copy pattern from the system tests and just adapt specific label to it.
```

Comment:
Maybe people unfamiliar with Testing Farm would benefit from a little context here? The purpose and setup.

Reply:
Updated :)